packages/mui-base/src/MultiSelectUnstyled/MultiSelectUnstyled.tsx

```diff
@@ -1,351 +0,0 @@
-import * as React from 'react';
```

Comment:
Instead of deleting the file, let's throw an error with a helpful message on how to migrate.

Possibly similar to https://github.com/mui/material-ui/issues/35501, even though developers do not import the deleted modules.

Reply:
We had the same thought :) https://github.com/mui/material-ui/pull/36274#pullrequestreview-1308721150